soc/xtensa/intel_adsp/ace_v1x/xtensa/config/tie-asm.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2021 Intel Corporation
```

Comment:
This single xtensa-config file is still include. On purpose?

Reply:
Yeah, check comments content of the header.